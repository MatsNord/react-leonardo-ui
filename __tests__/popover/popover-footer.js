import React from 'react';
import renderer from 'react-test-renderer';

import Popover from '../../src/popover';

test('Popover footer renders correctly', () => {
  const tree = renderer.create(
    <Popover.Footer>Footer content</Popover.Footer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Popover footer passes props correctly', () => {
  const tree = renderer.create(
    <Popover.Footer className="my-class" data-x="x" title="Footer title">
      Footer title
    </Popover.Footer>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
