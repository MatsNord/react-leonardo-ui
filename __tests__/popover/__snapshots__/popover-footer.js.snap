// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Popover footer passes props correctly 1`] = `
<div
  className="lui-popover__footer my-class"
  data-x="x"
  title="Footer title"
>
  Footer title
</div>
`;

exports[`Popover footer renders correctly 1`] = `
<div
  className="lui-popover__footer"
>
  Footer content
</div>
`;
